src/Compilation.zig

```diff
@@ -342,11 +342,18 @@ pub const AllErrors = struct {
             /// Does not include the trailing newline.
             source_line: ?[]const u8,
             notes: []Message = &.{},
+            reference_trace: []Message = &.{},
         },
         plain: struct {
             msg: []const u8,
             notes: []Message = &.{},
         },
+        ref: struct {
```

Comment:
Any particular reason this is another tagged union field rather than a separate, named struct, and then having `reference_trace: []Ref` rather than `reference_trace: []Message` ?

Reply:
No, that would require less changes so I'm not sure why I didn't do that originally.